schema/note.mod

```diff
@@ -1064,39 +1064,40 @@
 <!ELEMENT articulations
 	((accent | strong-accent | staccato | tenuto |
 	  detached-legato | staccatissimo | spiccato |
-	  scoop | plop | doit | falloff | breath-mark | 
-	  caesura | stress | unstress | other-articulation)*)>
+	  scoop | plop | doit | falloff | breath-mark |
+	  caesura | stress | unstress | soft-accent | 
+	  other-articulation)*)>

```

Comment:
We also need to add a soft-accent element definition within the note.mod DTD. It can be copied from the accent definition, using the same documentation as in the XSD. The DTDs have been keeping to a maximum 65 character line length so the documentation will wrap, unlike the XSD.